articles/machine-learning/how-to-prepare-datasets-for-automl-images.md

```diff
@@ -76,6 +80,35 @@ az ml data create -f [PATH_TO_YML_FILE] --workspace-name [YOUR_AZURE_WORKSPACE]
 [!Notebook-python[] (~/azureml-examples-main/sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb?name=upload-data)]
 ---
 
+If you already have your data present in Azure Blob Storage and want to create data asset out of it, you can do so by providing path to the location in Azure Blob Storage as shown below.
+
```

Comment:
Should the below code snippets have code to create the data asset - "az ml data create" for CLI and "create_or_update" for SDK ?

Reply:
I thought it'd be just repetition. Hence, was thinking to only  highlight the change. Do you think, we should write the whole code snippet again ?

Reply:
It may not be very clear to the reader, that they have to create a data asset again. Maybe add a note that they have to do this and point to above section.

Reply:
done

Reply:
Thanks Vivek. The changes look good now :)